src/klein/app.py

```diff
@@ -175,8 +182,9 @@ def route(self, url, *args, **kwargs):
             def index(request):
                 return "Hello"
 
-        @param url: A werkzeug URL pattern given to C{werkzeug.routing.Rule}.
-        @type url: str
+        @param url: A C{twisted.python.url.URL} instance, transformed to werkzeug
+            pattern, and thus used to initialize C{werkzeug.routing.Rule}.
```

Comment:
I think the prior `@param` string was fine.  We know from the `@type` string below that it's a `URL`, so there's no need to note that here.